src/getBibCodeFromObject.js

```diff
@@ -122,7 +122,7 @@ function fieldsAreOk(object) {
 	// ========== All fields are ok ['entryType', 'key', 'data'] ==========
 	// Now let's move to check for valid fields for any specific entry type
 	const entryType = object.entryType; // Checks entry type etc.
-	const bibtex = require('./BibTeX');
```

Comment:
BibTeX should be spelled with captial X (see https://en.wikipedia.org/wiki/BibTeX). Thus, this fix seems to be a quick fix. Could you fix it in the other way so taht this require statement works?